src/ServiceManagement/Compute/Commands.ServiceManagement.Extensions.Test/Commands.ServiceManagement.Extensions.Test.csproj

```diff
@@ -107,19 +118,29 @@
   <ItemGroup>
     <Compile Include="DSC\DscExtensionConfigurationParsingHelperTests.cs" />
     <Compile Include="DSC\DscExtensionSettingsSerializerTests.cs" />
+    <Compile Include="DSC\UnitTests\GetAzureVmDscExtensionStatusUnitTest.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\..\Common\Commands.Common\Commands.Common.csproj">
+      <Project>{5EE72C53-1720-4309-B54B-5FB79703195F}</Project>
+      <Name>Commands.Common</Name>
+    </ProjectReference>
     <ProjectReference Include="..\..\..\Common\Commands.ScenarioTests.Common\Commands.ScenarioTests.Common.csproj">
       <Project>{C1BDA476-A5CC-4394-914D-48B0EC31A710}</Project>
       <Name>Commands.ScenarioTests.Common</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\Services\Commands.Utilities\Commands.Utilities.csproj">
+      <Project>{4900EC4E-8DEB-4412-9108-0BC52F81D457}</Project>
+      <Name>Commands.Utilities</Name>
+    </ProjectReference>
     <ProjectReference Include="..\Commands.ServiceManagement\Commands.ServiceManagement.csproj">
       <Project>{e1ca72ba-8374-45f6-904d-fd34ecdf5b6f}</Project>
       <Name>Commands.ServiceManagement</Name>
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <None Include="MSSharedLibKey.snk" />
```

Comment:
Spoke with Yugang offline - once there is a common place for the .snk file all duplicates will be removed. We thought about putting a duplicate .snk file but decided to go with the pattern that is followed so far ( to be consistent). Ideally, yes there should be a common place for the .snk file.